src/avrdude.conf.in

```diff
@@ -1458,6 +1458,23 @@ programmer
     usbpid                 = 0x0c9f;
 ;
 
+#------------------------------------------------------------
+# gemmaBoot
+#------------------------------------------------------------
+
+# Serving the Gemma bootloader
+# https://github.com/adafruit/Adafruit-Trinket-Gemma-Bootloader
+
+programmer
+    id                     = "gemmaBoot";
```

Comment:
Thanks for the PR! perhaps the id should be `"gemmaboot"` with a lowercase B instead? It looks like most programmers in avrdude.conf have id's in lowercase letters

Reply:
It is better to be consistent and use `gemmaboot`.